flink-autoscaler-plugin-jdbc/src/main/java/org/apache/flink/autoscaler/jdbc/event/JdbcEventInteractor.java

```diff
@@ -92,12 +94,23 @@ public void createEvent(
             String message,
             String eventKey)
             throws Exception {
+        createEvent(jobKey, reason, type, message, eventKey, Timestamp.from(clock.instant()));
+    }
+
+    @VisibleForTesting
+    void createEvent(
+            String jobKey,
+            String reason,
+            AutoScalerEventHandler.Type type,
+            String message,
+            String eventKey,
+            Timestamp createTime)
+            throws Exception {
```

Comment:
Don't need introduce new method for testing, you could setClock to control the timestamp.

Check `AbstractJdbcAutoscalerEventHandlerITCase#testEventIntervalWithoutMessageKey`